src/webgpu/shader/execution/expression/expression.ts

```diff
@@ -132,6 +132,26 @@ export interface ExpressionBuilder {
   (values: Array<string>): string;
 }
 
+// A Pipeline is a map of WGSL shader source to a built pipeline
```

Comment:
Normally the content of a WebGPU pipeline also depends on override settings.
But since these tests don't use overrides, this is sufficient.

Reply:
Sure. The scoping of this cache is fine for the single-test-case, and for the test types we currently have.